src/definitions/elements/button.less

```diff
@@ -897,11 +904,8 @@
     .ui.tertiary.button {
         transition: color @defaultDuration @defaultEasing !important;
         border-radius: 0;
-        margin:
-            (@verticalPadding - @tertiaryVerticalPadding)
-            (@horizontalMargin)
-            (@verticalPadding + @shadowOffset + @verticalMargin - @tertiaryVerticalPadding)
-            0 !important;
+        margin: (@verticalPadding - @tertiaryVerticalPadding) (@horizontalMargin)
```

Comment:
I actually find the previous formatting more readable than the new one tbh

Reply:
me too, but Prettier is not a linter, and cannot be configured, as long as Stylelint requires Prettier, we cannot do much except patching Prettier more, which is not easy I would like to avoid more patches than really needed

Reply:
Well, then we should really think twice, if we really want to use prettier then, if it causes more trouble than help 🤔

Reply:
eslint is perfect, no need for Prettier fo JS

but stylelint for LESS is quite buggy, it does not detect a lot of formatting problems, so I would like to find a solution how to integrate with Prettier, not mentioning next release of Stylelint will rely on Prettier solely